woocommerce/wc-admin
# Add in Razorpay as default fallback payment gateway

Fixes (part of) woocommerce/woocommerce#32258 

* Adds in the gateway to the default list.
* Fixes up some translations in alt text for other gateways.

### Screenshots

<img width="689" alt="Screen Shot 2021-06-01 at 4 26 28 PM" src="https://user-images.githubusercontent.com/10561050/120386382-8be95480-c2f6-11eb-8f88-d6f058e05b9e.png">


### Detailed test instructions:

1. Clone this branch of Razorpay to `woo-razorpay` in your WP plugins folder - https://github.com/razorpay/razorpay-woocommerce/pull/195
2. Delete any remote payment transients and deactivate the remote payment tester if active.
3. Set your store country to India and don't select CBD industry during onboarding.
2. Go to the payments task.
3. Note that razorpay exists in the list.
4. If the plugin from step 1 is correctly copied to your plugin's folder, note that the setup fields are shown.
5. Complete the setup and note that settings persist and Razorpay is moved to the completed gateway list.
